### PR TITLE
Order resources on web/cli

### PR DESCRIPTION
**Pull request by bliuredhat**
Order resources on web/cli.
To make an order for API is sorted by its first letter .The way is to rewrite the get_api_root_view() function in /contrib/bulk_operations/bulk_operations.py doc.

### CODE REVIEW

**Comment by lubomir** [comment]
The implementation looks fine to me. However, I think the files from your IDE should not be part of this commit. There is also an extra file called `out.html`.

Why the change in requirements? In my opinion it should be at least mentioned in the commit message or be separate commit altogether.

The merge conflict is caused by the change in settings. This commit should not change the settings as the change is unrelated to this task.

I don't think this should be part of `BulkRouter`, as it is not relevant to bulk operations. It might be better to create `PDCRouter` (or any better name) in `pdc/routers.py` (or maybe completely separate file) which would inherit from `BulkRouter` and implement this feature.

**Comment by bliuredhat** [comment]
lubomir, Thank you so much. Your opinion is right and it's easy to edit.

**Comment by lubomir** [comment]
You were right about changing the dependency. That will fix the build failure. Just mention it in commit message that the the router does not work with REST Framework 3.2.

**Comment by bliuredhat** [comment]
lubomir , thank you for your help, it's good for me.

**Comment by lubomir** [comment]
Looks great!

Since we already depend on Django >= 1.8.1, you can remove the workaround for older version.
If you want to, you could even write a simple test that will get API root and verify that the keys are really sorted (there is no need to check if all registered views are included). This could be in a separate commit and pull request and does not need to block this one.

**Comment by bliuredhat** [comment]
lubomir, it sounds cool !

**Comment by erichuanggit** [comment]
I would suggest moving the changes into a single file as what `BulkRouter` is, keeping `routers.py` tidy.

**Comment by bliuredhat** [comment]
erichuanggit, It's cool ideal , thank you and I will do it.

**Comment by lubomir** [comment]
Maybe `pdc/apps/utils/routers.py` would be better name. It would change the import into `from pdc.apps.utils.routers import PDCRouter`.

**Comment by xychu** [comment]
+1

**Comment by erichuanggit** [comment]
How about `SortedRouter.py`? `BulkRouter` looks like not making sense.

**Comment by bliuredhat** [comment]
erichuanggit, lubomir ; Okay, it's a good suggest for me, and I will to do now.

File: pdc/apps/utils/BulkRouter.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
```
Comment (erichuanggit) [inline pdc/apps/utils/BulkRouter.py]: `BulkRouter.py` this file is not required.

**Pull request closed without merging**